content/contribute/add_phone_to_plugin.md

```diff
@@ -0,0 +1,234 @@
+in question is located in the `tftpboot` directory in the installed plugin, located at
+`/var/lib/wazo-provd/plugins/wazo-yealink-v85/var/tftpboot/firmware`. It is possible to have more
+than one step in the install method. To see how it is done, look at what is done with the language
+packages already present in the `pkgs.db` file.
+
+The *file definition* is usually at the bottom of the `pkgs.db` file:
+
+```
+[file_T46S_T48S_T42S_T41S-fw]
+filename: T46S(T48S,T42S,T41S)-66.85.0.5.rom
+url: http://support.yealink.com/forward2download?path=ZIjHOJbWuW/DFrGTLnGypkTJASVqwzX/x71yaViKhXSKOgYSKl0Pb7N9IXFpYf7YAOhOJmV0fGjbcYgmf0b9uxGdQ1IFTWkfgTz2m4g5Uoy1NWm3/S6JBTsplusSymbolM5UOVemCq9Kf2pvmyoc=
+size: 25559152
+sha1sum: 091d469ba3ce13b424a5212fb74f0e1b59c31a6a
+```
+
+The size is in bytes and the sha1sum is obtained using the `sha1sum` command on a GNU/Linux system.
+
+### `plugins/xivo-yealink-v85/templates/T41S.tpl`
+
+It is necessary to create a `T41S.tpl` file or copy-paste an existing file and rename it to
+`T41S.tpl`.Make sure that its content is correct.
+
+In our case it should be:
+
+```jinja
+{% extends 'base.tpl' -%}
+
+{% block model_specific_parameters -%}
+gui_lang.url = http://{{ ip }}:{{ http_port }}/lang/T41S-T42S-T53W-T53/004.GUI.French.lang
+{% endblock %}
+
+```
+
+### `plugins/xivo-yealink-v85/plugin-info`
+
+This file is for metadata. It is used to expose what phones are supported by the plugin, the plugin
+version and the features supported by the phones and the plugin.
+
+In our example case, we need to:
+
+1. Add the model to the `description` and to the French translation `description_fr`. In our case,
+    we decide to specify that we support the T4X series.
+
+    ```json
+    "description": "Plugin for Yealink for CP920, CP960, T2X, T3X, T4X, T5X and W60 series in version V85.",
+    "description_fr": "Greffon pour Yealink pour les series CP920, CP960, T2X, T3X, T4X, T5X et W60 en version V85.",
+    ```
+
+2. Add the phone model in the `capabilities` key
+
+    ```json
+    "Yealink, T41S, 66.85.0.5": {
+        "lines": 6,
+        "high_availability": true,
+        "function_keys": 15,
+        "expansion_modules": 0,
+        "protocol": "sip"
+    },
+    ```
+
+    As you can see, we reuse the information that we defined in the `common.py` file. To see the
+    supported metadata, take a look at the
+    [README.md](https://github.com/wazo-platform/wazo-provd-plugins/blob/master/README.md) file at
+    the root of the wazo-provd-plugins repository.
+
+3. Bump the plugin version
+
+    It is necessary for the plugin version to be incremented, otherwise the plugin will not be
+    rebuilt one merged into the main branch. To do so, increment the last digit of the `version`
+    key.
+
+## Final steps
+
+Commit everything you have done in git with the format `plugin-name: changes` as the commit
+message. For example, `yealink-v85: add T41S support`.
+
+You are now ready to open the pull request on GitHub.
```

Comment:
Add a link to the GitHub pull request https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request